db/seeds.rb

```diff
@@ -13,3 +14,17 @@
 ActiveRecord::Base.connection.reset_pk_sequence!('user_likes')
 ActiveRecord::Base.connection.reset_pk_sequence!('posts')
 ActiveRecord::Base.connection.reset_pk_sequence!('users')
+
+
+100.times do
+  User.create!(name:"#{Faker::Games::WarhammerFantasy.hero.split(' ')[0]} #{Faker::Science.scientist.split(' ')[0]} #{Faker::FunnyName.two_word_name.split(' ')[1]}", email: Faker::Internet.email , password: "badpassword", password_confirmation: "badpassword")
+end
+
+100.times do
+  Post.create!(content: Faker::GreekPhilosophers.quote, only_followers: [true,false].sample, user_id: rand(1..100))
+end
+
+100.times do
+  UserLike.create!(user_id: rand(1..100).to_s, post_id: rand(1..100).to_s)
+  FollowerFollowed.create!(follower_id: rand(1..100).to_s, followed_id: rand(1..100).to_s)
+end
```

Comment:
Now at first one might think that technically this will create some likes that might not normally be able to be created, because you can't like a post that you can't see because you don't follow a person.  But at this point there is no code that removes a like from a post if you unfollow someone, so we can just imagine these likes occurred with the user was following the other user, and then unfollowed them later